subscribers:

  camera_reading:
    topic: /camera/image
    queue_size: 1

actions:

  camera_reading:
    name: /ssd_ros/check_for_objects

publishers:

  seg_image:
    topic: /seg_ros/seg_image

  object_detector:
    topic: /ssd_ros/found_object
    queue_size: 1
    latch: false

  bounding_boxes:
    topic: /ssd_ros/bounding_boxes
    queue_size: 1
    latch: false

  detection_image:
    topic: /ssd_ros/detection_image
    queue_size: 1
    latch: true

image_view:

  enable_opencv: fasle
  wait_key_delay: 1
  enable_console_output: true
